resolwe/process/runtime.py

```diff
@@ -13,6 +13,14 @@
 # Outputs class name.
 PROCESS_OUTPUTS_NAME = "Output"
 
+try:
+    # If the import suceeds we are running on the server.
+    from resolwe.flow.execution_engines.python.python_process import PythonProcess
+
+except ModuleNotFoundError as e:
+    # We are in the container, the PythonProcess class in unavailable.
+    PythonProcess = object
```

Comment:
This is a bit ugly, but you have to do something like this at one point.

Reply:
Actually I could move the entire PythonProcess class and plugin manager into `resolwe.process` module. It would actually fit quite nicely inside there.